Cython/Debugger/Cygdb.py

```diff
@@ -104,42 +104,38 @@ def make_command_file(path_to_debug_info, prefix_code='',
 
     return tempfilename
 
-usage = "Usage: cygdb [options] [PATH [-- GDB_ARGUMENTS]]"
 
-def main(path_to_debug_info=None, gdb_argv=None, no_import=False):
+def main():
     """
     Start the Cython debugger. This tells gdb to import the Cython and Python
     extensions (libcython.py and libpython.py) and it enables gdb's pending
     breakpoints.
-
-    path_to_debug_info is the path to the Cython build directory
-    gdb_argv is the list of options to gdb
-    no_import tells cygdb whether it should import debug information
     """
-    parser = optparse.OptionParser(usage=usage)
-    parser.add_option("--gdb-executable",
+    parser = argparse.ArgumentParser(
+        prog="cygdb",
+        description="Cython debugger",
+    )
+    parser.add_argument("gdb_argv", nargs="*",
+                        help="arguments to forward to gdb; specified after --")
+    parser.add_argument("--build_dir", default=os.curdir,
+                        help="directory containing cython_build/ files")
+    parser.add_argument("--no_import", default=False,
```

Comment:
I am thinking about this argument. Do we need it? I think that because `--no_import` should be used only and only when `--build_dir` is not specified and vise versa. 

I propose just to set `no_import` to `True` when `--build-dir` is not specified.

This will avoid code managing the case when both arguments are specified (PR currently does not manages it). Moreover, current master version has same semantics.